schema/vrs-source.yaml

```diff
@@ -184,26 +184,31 @@ definitions:
     additionalProperties: false
     type: object
     description: >-
-      The count of discrete copies of a Feature, Molecular Variation, or
-      other molecule within a genome.
+      The count of discrete copies of a subject :ref:`MolecularVariation`,
```

Comment:
Should this be re-written as...

> The 'absolute' count of discrete copies of a MolecularVariation, Feature, SequenceExpression or CURIE reference contained within a system (i.e. genome, cell, etc...).

I'm not sure the current phrasing is clear. What does 'to any of these' mean in the context of count of discrete copies of a subject...?